OMEdit/OMEditLIB/Modeling/LibraryTreeWidget.cpp

```diff
@@ -3351,15 +3399,19 @@ void LibraryTreeView::showContextMenu(QPoint point)
           if (pLibraryTreeItem->isTopLevel()) {
             mpUnloadClassAction->setText(Helper::unloadClass);
             mpUnloadClassAction->setStatusTip(Helper::unloadClassTip);
+            mpReloadClassAction->setText(Helper::reloadClass);
+            mpReloadClassAction->setStatusTip(Helper::reloadClassTip);
           } else {
             mpUnloadClassAction->setText(Helper::deleteStr);
             mpUnloadClassAction->setStatusTip(tr("Deletes the Modelica class"));
           }
           // only add unload/delete option for top level system libraries
           if (!pLibraryTreeItem->isSystemLibrary()) {
             menu.addAction(mpUnloadClassAction);
+            menu.addAction(mpReloadClassAction);
```

Comment:
This will add reload menu to nested classes as well. If you intend to add reload just to top-level then you need to re-write this block.